pkg/render/logstorage/linseed/linseed.go

```diff
@@ -308,6 +253,60 @@ func (l *linseed) linseedDeployment() *appsv1.Deployment {
 	}
 }
 
+func (l *linseed) volumeMounts() []corev1.VolumeMount {
```

Comment:
I'd rather we didn't split these out into separate functions - these are only used once, so there's no need to make a common function for it. The code is easier to read if all of this lives within `linseedDeployment`.

Same goes for `envVars` and `container`

Reply:
Sounds good to me!